website_form_builder/static/src/js/widgets.js

```diff
@@ -0,0 +1,166 @@
+/* Copyright 2017 Tecnativa - Jairo Llopis
+ * License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+
+odoo.define('website_form_builder.widgets', function(require){
+    "use strict";
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+    var Model = require('web.Model');
+    var options = require('web_editor.snippets.options');
+    var widget = require("web_editor.widget");
+    var _t = core._t;
+    var Dialog = widget.Dialog;
+
+    core.qweb.add_template("/website_form_builder/static/src/xml/widgets.xml");
+
+    return {
+        DefaultValueForm: Dialog.extend({
+            template: "website_form_builder.DefaultValueForm",
+
+            /**
+             * Store needed field information.
+             *
+             * @param {DOMElement} $field DOM element of the field
+             * that wants a new default value.
+             */
+            init: function (parent, options, field) {
+                this.field_html = $(field).html();
+                options = $.extend({}, {
+                    title: _t("Set field's default value"),
+                    size: "small",
```

Comment:
trailing comma no bueno in JS

Reply:
I'd say it's safe to use it nowadays. 🤔 See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#Browser_compatibility